yomi.lua

```diff
@@ -25,7 +25,13 @@ local function yomi_config(opts)
     name = N,
     url = "",
     timeout = 5.0,
+    log_virus = true,
+    log_suspicious = true,
+    log_virus = true,
+    log_unsure = true,
```

Comment:
Just a better naming convention, default to false:
```suggestion
    log_unknown = false,
```